src/storage/index/IndexPolicyMaker.h

```diff
@@ -0,0 +1,117 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#ifndef STORAGE_INDEXPOLICYMAKER_H
+#define STORAGE_INDEXPOLICYMAKER_H
+#include "base/Base.h"
+#include "meta/SchemaManager.h"
+#include "storage/CommonUtils.h"
+#include "storage/index/IndexOptimizer.h"
+
+namespace nebula {
+namespace storage {
+
+enum class PolicyType : uint8_t {
+    /**
+     * Not optimized, PolicyScanType will be invalid here. default using SEEK_SCAN.
+     */
+    SIMPLE_POLICY      = 0x01,
+    /**
+     * Scan policy optimized,  PolicyScanType will be valid here.
+     */
+    OPTIMIZED_POLICY   = 0x02,
+};
+
+enum class PolicyScanType : uint8_t {
+    /**
+     * No index column match. seek scan and using expression condition filter.
+     */
+    SEEK_SCAN          = 0x01,
+    /**
+     * Few index column match, index scan in subset.
+     * And using expression condition filter for remaining index column.s
+     */
+    PREFIX_SCAN        = 0x02,
```

Comment:
PREFIX_SCAN and ACCURATE_SCAN are the same things.  Unify them.

Reply:
They are different, `ACCURATE_SCAN` does not need to use expression filtering ,  PREFIX_SCAN and SEEK_SCAN needed. I will unify PREFIX_SCAN and SEEK_SCAN . 
WDYT?